src/group_theory/group_action/basic.lean

```diff
@@ -99,10 +136,8 @@ variables (α β) [add_monoid α] [add_action α β]
 class is_pretransitive : Prop :=
 (exists_vadd_eq : ∀ x y : β, ∃ g : α, g +ᵥ x = y)
 
-variables {β}
-
-lemma exists_vadd_eq [is_pretransitive α β] (x y : β) :
-  ∃ m : α, m +ᵥ x = y := is_pretransitive.exists_vadd_eq x y
+attribute [to_additive] mul_action.is_pretransitive mul_action.exists_smul_eq
+  mul_action.surjective_smul mul_action.orbit_eq_univ mul_action.regular.is_pretransitive
```

Comment:
Can you do this with `@[]` above instead? Doing it this way makes it easy to miss one.

Reply:
Done.